data/feeds.csv

```diff
@@ -31819,6 +31819,7 @@ TV5Linares.cl,SD,SD,,TRUE,c/CL,America/Santiago,spa,480i
 TV5Monde.fr,SD,SD,,TRUE,c/FR,Europe/Paris,fra,576i
 TV5MondeAfrica.fr,SD,SD,,TRUE,r/AFR,Europe/Paris,fra,576i
 TV5MondeAsia.fr,SD,SD,,TRUE,r/SEA,Europe/Paris,fra,576i
+TV5MondeEurope.fr,Netherlands,Netherlands,,FALSE,c/NL,Europe/Amsterdam,fra,1080i
 TV5MondeEurope.fr,SD,SD,,TRUE,r/EUR,Europe/Paris,fra,576i
 TV5MondeFranceBelgiumSwitzerlandMonaco.fr,HD,HD,,FALSE,c/BE;c/CH;c/FR;c/MN,Europe/Paris,fra,1080i
 TV5MondeFranceBelgiumSwitzerlandMonaco.fr,SD,SD,,TRUE,c/BE;c/CH;c/FR;c/MN,Europe/Paris,fra,576i
```

Comment:
I also suspect `TV5MondeFranceBelgiumSwitzerlandMonaco.fr` is a duplicate of `TV5Monde.fr`. Does anyone have more insight in this?